### PR TITLE
WIP Adds ability to delete youth pics

### PR DESCRIPTION
**Pull request by Unesdala**

### CODE REVIEW

**Comment by Unesdala** [comment]
@JoshuaVSherman Mind helping to figure out a better way of linking the drop down to makeDropDown?

I think it needs to be able to link it via the individual ID's but passing along both and trying to tie it to another state variable of _id that's linked to the two ID's we currently have doesn't work.

Something similar will probably be needed for the delete validation. Right now, either one being filled validates both buttons. Clicking on the wrong one deletes everything instead of what was passed along to validate it which was...an interesting bug to find.

**Comment by JoshuaVSherman** [comment]
@Unesdala just follow what I did to for deleting the forums, it is basically the same thing here, ping me if you need more help after looking at the code that deletes the forums, thanks

File: src/containers/AdminDashboard/index.js [hunk 1]
```diff
@@ -177,9 +195,23 @@ export class AdminDashboard extends Component {
     return true;
   }
 
-  validateDelete() {
+  validateDeleteForum() {
     const { forumId } = this.state;
-    if (forumId !== '') return false;
+    let notEmpty = false;
```
Comment (JoshuaVSherman) [inline src/containers/AdminDashboard/index.js]: @Unesdala this code is no better than how it was originally, just adds more variables, please put it back how it was before

File: src/containers/AdminDashboard/index.js [hunk 2]
```diff
@@ -177,9 +195,23 @@ export class AdminDashboard extends Component {
...
+    if (notEmpty) {
+      return false;
+    }
+    return true;
+  }
+
+  validateDeleteYouth() {
+    const { youthPicsId } = this.state;
+    let notEmpty = false;
+    if (youthPicsId !== '') notEmpty = true;
```
Comment (JoshuaVSherman) [inline src/containers/AdminDashboard/index.js]: @Unesdala you just need to return false here, and get rid of the extra variables please

File: src/lib/forms.js [hunk 3]
```diff
@@ -12,6 +12,19 @@ const makeDropdown = (htmlFor, labelText, value, onChange, options, oValue, dVal
     </select>
   </label>
 );
+
+const makeYouthDropdown = (htmlFor, labelText, value, onChange, options, oValue, dValue) => (
+  <label htmlFor={htmlFor} style={{ paddingTop: '12px' }} id={htmlFor}>
+    {labelText}
+    <br />
+    <select id={htmlFor} value={value} onChange={(event) => onChange(event, 'youthPicsId')}>
+      <option id="blank-option" key="blank-option" value="">---</option>
```
Comment (JoshuaVSherman) [inline src/lib/forms.js]: @Unesdala you just need to use the same function and use the htmlFor to be the state variable, then replace it in the onChange where the state variable is there in single quotes right now, please fix for both delete dropdowns and remove the second function that is almost a complete identical one

File: src/containers/AdminDashboard/index.js [hunk 4]
```diff
@@ -177,9 +211,23 @@ export class AdminDashboard extends Component {
...
   }
 
-  validateDelete() {
+  validateDeleteForum() {
     const { forumId } = this.state;
-    if (forumId !== '') return false;
+    let notEmpty = false;
+    if (forumId !== '') notEmpty = true;
+    if (notEmpty) {
+      return false;
```
Comment (JoshuaVSherman) [inline src/containers/AdminDashboard/index.js]: @Unesdala please fix here too

File: src/lib/forms.js [hunk 5]
```diff
@@ -12,6 +12,19 @@ const makeDropdown = (htmlFor, labelText, value, onChange, options, oValue, dVal
     </select>
   </label>
 );
+
+const makeYouthDropdown = (htmlFor, labelText, value, onChange, options, oValue, dValue) => (
+  <label htmlFor={htmlFor} style={{ paddingTop: '12px' }} id={htmlFor}>
+    {labelText}
+    <br />
+    <select id={htmlFor} value={value} onChange={(event) => onChange(event, 'youthPicsId')}>
```
Comment (JoshuaVSherman) [inline src/lib/forms.js]: @Unesdala so where it has 'youthPicsId' on line 20, you would pass in the htmlFor parameter, and set that to be youthPicsId, if you can't figure it out, i'll finish it up in the morning though no problem

**Pull request merged**